### PR TITLE
Fix Multichain aggregation SQL queries

### PR DESCRIPTION
**Pull request by Captain-Coder**
Relying on MAX() to leave the right row under the cursor for non-aggregate query results is explicitly not guaranteed to work in sqlite.

### CODE REVIEW

**Comment by Captain-Coder** [comment]
Retest this please.
Tests failed on Tribler.Test.Core.Modules.RestApi.test_settings_endpoint.TestSettingsEndpoint.test_get_settings which seems unrelated

**Comment by brussee** [comment]
Reviewed changes, looks good to me.

File: Tribler/community/multichain/database.py [hunk 1]
```diff
@@ -232,7 +230,7 @@ def contains(self, hash_requester):
         :param hash_requester: The hash_requester that is queried
         :return: True if the block exists, else false.
         """
-        db_query = u"SELECT hash_requester FROM multi_chain WHERE hash_requester == ? LIMIT 1"
+        db_query = u"SELECT hash_requester FROM multi_chain WHERE hash_requester = ? LIMIT 1"
         db_result = self.execute(db_query, (buffer(hash_requester),)).fetchone()
```
Comment (brussee) [inline Tribler/community/multichain/database.py]: Why do you need a buffer?

Comment (Captain-Coder) [inline Tribler/community/multichain/database.py]: The python-sqlite module requires that all BLOB type parameters are presented as python type buffer().

**Pull request merged**